app/router.js

```diff
@@ -41,9 +41,7 @@ AppRouter.map(function() {
     });
   });
 
-  this.route('project', {
-    path: '/:slugged_route_slug/:project_slug'
-  }, function() {
+  this.route('project', function() {
```

Comment:
Yeah, this was just for rendering the page. Didn't want to go through adding projects just to visually look at the component. I can leave a `TODO` here so it is more clear but this will be fixed when this is ready to be merged.